### PR TITLE
Changed spawn code on respawn to PFH format

### PR DESCRIPTION
**Pull request by nikolauska**

### CODE REVIEW

File: addons/respawn/functions/fnc_module.sqf [hunk 1]
```diff
@@ -33,15 +33,19 @@ GVAR(Module) = true;
...
                     deleteVehicle _unit;
-                };
-            };
+
+                    [_this select 0] call CBA_fnc_removePerFrameHandler;
+                  };
+                },
+                4,
+                [_this]
+            ] call CBA_fnc_addPerFrameHandler
```
Comment (CorruptedHeart) [inline addons/respawn/functions/fnc_module.sqf]: You will probably want to change this to [FUNC(common,waitAndExecute)](https://github.com/acemod/ACE3/blob/master/addons/common/functions/fnc_waitAndExecute.sqf). Delayed CBA_fnc_addPerFrameHandler still executes the first time it runs. I believe.

File: addons/respawn/functions/fnc_removeBody.sqf [hunk 2]
```diff
@@ -29,9 +29,12 @@ if (_forceRemove) then {
...
-};
+[{
+
+  _unit = (_this select 1) select 0;
+  // hideBody takes ~20s till body is fully underground
+  // a better hideBody would make this more aesthetic
+  deleteVehicle _unit;
+},
+_bodyRemoveTime,
+[_body]] call CBA_fnc_addPerFrameHandler
```
Comment (CorruptedHeart) [inline addons/respawn/functions/fnc_removeBody.sqf]: Same for here

File: addons/respawn/functions/fnc_module.sqf (hunk 1 above)
Comment (esteldunedain) [inline addons/respawn/functions/fnc_module.sqf]: Yes, that's right. This code will remove the body inmediately

**Comment by nikolauska** [comment]
Ok, I'll fix it to use FUNC(common,waitAndExecute)

**Comment by Glowbal** [comment]
> Ok, I'll fix it to use FUNC(common,waitAndExecute)

This is the correct macro: `EFUNC(common,waitAndExecute)`

**Comment by jaynus** [comment]
Needs to use proper 4-space tabbing.

**Pull request merged**